### ISSUE TITLE
Should I dispatch the load event on the cloned node by myself?

### ISSUE THREAD

**Issue by Alex1990** [eKoopmans/html2pdf#67]
The below code bound the `load` event.

https://github.com/eKoopmans/html2pdf/blob/dbce51e917e32859cfb3890f0e4ec4e4e6d15eb7/src/utils.js#L52-L55

**Comment by eKoopmans** [comment]
Hi @Alex1990, the load event should be triggered automatically when the node is attached to a document, i.e.:
https://github.com/eKoopmans/html2pdf/blob/master/src/index.js#L124-L126

Is there an issue with it not firing?

**Comment by Alex1990** [comment]
Yes. It's not triggered. I must trigger it manually.

**Comment by eKoopmans** [comment]
Okay, sounds like a problem. Can you provide an example? A Fiddle would be useful (see [docs](https://github.com/eKoopmans/html2pdf#issues)). Thanks!

**Comment by Alex1990** [comment]
I just wonder where it states the `load` event will be triggered after appending the node to the document.

**Comment by eKoopmans** [comment]
Fair enough, [this example](https://jsfiddle.net/0y5ofmjm/1/) looks like it's not firing the load event. I had borrowed `cloneNode` from html2canvas. I will investigate more tomorrow. Thanks!
